internal/pkg/plugin/binary.go

```diff
@@ -149,6 +182,17 @@ func loadMetaByName(name, plugindir string) (*Meta, error) {
 	return m, nil
 }
 
+func loadMetaByFilename(filename string) (*Meta, error) {
+	fh, err := os.Open(filename)
+	if err != nil {
+		sylog.Debugf("Error opening meta file %q: %s\n", filename, err)
```

Comment:
`sylog.Errorf`

Reply:
I don't actually want that. The debug entry is there just in case it's needed (you know, for debugging :-) but for the user I would like to have a different error. That's why I'm checking whether it's a not exist error elsewhere.